src/intro-to-c/enums/enum-tutorial.md

```diff
@@ -0,0 +1,168 @@
+
+## Using Enums in Code
+
+When using enums, we must define a variable type. This can be done in one of two ways. I will reuse the `exp_value` example.
+
+```c
+enum exp_value {wheat = 2, onion, carrot, corn = 7, potatoes, strawberries = 11};
+enum exp_value crops;
+
+// or
+
+enum exp_value {wheat = 2, onion, carrot, corn = 7, potatoes, strawberries = 11} crops;
+```
+
+We can even define multiple variables of type `exp_value` like this:
+
+```c
+enum exp_value {wheat = 2, onion, carrot, corn = 7, potatoes, strawberries = 11} plants;
+enum exp_value crops, dirt_growers;
+
+// or
+
+enum exp_value {wheat = 2, onion, carrot, corn = 7, potatoes, strawberries = 11} crops, plants, dirt_growers;
+```
+
+Now we can assign and use our enumerated values in code. Lets look at something simple.
+
+``` c
+#include <stdio.h>
+
+enum exp_value {wheat = 2, onion, carrot, corn = 7, potatoes, strawberries = 11} crops, plants, dirt_growers;
+
+int main(void)
+{
+	crops = onion;
+	plants = corn;
+	dirt_growers = potatoes;
+	printf("Onion Experience Gain: %d\n", crops);
+	printf("Corn Experience Gain: %d\n", plants);
+	printf("Potatoes Experience Gain: %d\n", dirt_growers);
+	return 0;
+}
+```
+
+Here we assign the value of `onion` to `crops`, which we know is 3. The expected output would be as follows:
+
+```
+Onion Experience Gain: 3
+Corn Experience Gain: 7
+Potatoes Experience Gain: 8
+```
+
+But what if we want to get a bit more tricky with our enums, like this maybe?
+
+``` c
+int x = 8
+crops Mystery_crop = x;
+```
+
+The compiler will warn you that it is expecting a semicolon after `crops` if you try to define an enumerator variable this way without the use of `typedef`.
+
+### Using `typedef` With Enums
+
+We can use the `typedef` keyword to make our enumerated list a bit more versatile. For example, we will be able to create an enum variable, rather than being restricted to the constants in the enumerated list. Let's look at this example.
+
+``` c
+#include <stdio.h>
+
+typedef enum exp_value {wheat = 2, onion, carrot, corn = 7, potatoes, strawberries = 11} crops;
+
+int main(void)
+{
+	int x = 9;
+	crops Mystery_crop = x;
+
+	switch(Mystery_crop)
+	{
+		case wheat:
+			printf("It is wheat.\n");
+			break;
+		case onion:
+		...
+		default:
+			printf("You have found a new crop. You gained %d experience.\n", Mystery_crop);
+			break;
+	}
+	return 0;
+}
+```
+
+In this case `x = 9` is hard coded, but you could also take in an input and check if it is a crop you've encountered before.
+
+## Enum Errors
+
+Enumerated constants can only appear in one enumerated list. For example, this code would produce a `replication of enumerator` error:
+
+``` c
+#include <stdio.h>
+
+enum days_of_the_week {sun = 1, mon, tues, wed, thur, fri, sat} day;
+enum celsetial_bodies {sun, moon, jupiter} body;
+
+int main(void)
+{
+	day = mon;
+	printf("%d\n", mon);
+	body = sun;
```

Comment:
Same note about creating global variables.

Reply:
Corrected this, and the enumerator syntax.